static/app/gettingStartedDocs/apple/apple-ios.tsx

```diff
@@ -41,207 +27,72 @@ export const steps = ({
-
-    SentrySDK.start { options in
-        options.dsn = "${dsn}"
-        options.debug = true // Enabled debug when first installing is always helpful${
-          hasPerformance
-            ? `
-
-        // Set tracesSampleRate to 1.0 to capture 100% of transactions for performance monitoring.
-        // We recommend adjusting this value in production.
-        options.tracesSampleRate = 1.0`
-            : ''
-        }${
-          hasProfiling
-            ? `
-        // Set profilesSampleRate to 1.0 to profile 100% of sampled transactions.
-        // We recommend adjusting this value in production
-        options.profilesSampleRate = 1.0`
-            : ''
-        }
-    }
-
-    return true
-}
-        `,
-      },
-      {
-        description: (
+        additionalInfo: (
           <p>
             {tct(
-              "When using SwiftUI and your app doesn't implement an app delegate, initialize the SDK within the [initializer: App conformer's initializer]:",
+              'Alternatively, you can also [manualSetupLink:set up the SDK manually].',
               {
-                initializer: (
-                  <ExternalLink href="https://developer.apple.com/documentation/swiftui/app/main()" />
+                manualSetupLink: (
+                  <ExternalLink href="https://docs.sentry.io/platforms/apple/guides/ios/manual-setup/" />
                 ),
+                stepsBelow: <strong />,
               }
             )}
           </p>
         ),
-        language: 'swift',
-        code: `
-import Sentry
-
-@main
-struct SwiftUIApp: App {
-    init() {
-        SentrySDK.start { options in
-            options.dsn = "${dsn}"
-            options.debug = true // Enabled debug when first installing is always helpful${
-              hasPerformance
-                ? `
-
-            // Set tracesSampleRate to 1.0 to capture 100% of transactions for performance monitoring.
-            // We recommend adjusting this value in production.
-            options.tracesSampleRate = 1.0`
-                : ''
-            }${
-              hasProfiling
-                ? `
-            // Set profilesSampleRate to 1.0 to profile 100% of sampled transactions.
-            // We recommend adjusting this value in production
-            options.profilesSampleRate = 1.0`
-                : ''
-            }
-        }
-    }
-}
-        `,
       },
     ],
   },
   {
     type: StepType.VERIFY,
-    description: (
-      <p>
-        {tct(
-          'This snippet contains an intentional error you can use to test that errors are uploaded to Sentry correctly. You can add it to your main [viewController: ViewController].',
-          {
-            viewController: <code />,
-          }
-        )}
-      </p>
+    description: t(
+      'The Sentry wizard automatically adds a code snippet that generates an error to your project. Simply run your app and you should see the error in your Sentry project.'
```

Comment:
To snippet we add is `SentrySDK.capture(message: "...")` instead of throwing an error.

```suggestion
      'The Sentry wizard automatically adds a code snippet that captures a message to your project. Simply run your app and you should see this message in your Sentry project.'
```